concepticondata/references/references.bib

```diff
@@ -3102,3 +3102,78 @@ @misc{Hsiu2015
   doi = {10.5281/zenodo.1127804},
   publisher={Center for Research in Computational Linguistics}
 }
+
+@article{Maciejewski2016,
+  title={{Relative meaning frequencies for 100 homonyms: British eDom norms}},
+  author={Maciejewski, Greg and Klepousniotou, Ekaterini},
+  journal={Journal of Open Psychology Data},
+  volume={4},
+  year={2016},
+  publisher={Ubiquity Press}
```

Comment:
This is a journal, right? So you can delete the publisher, but should try and find the page number, and and the journal number also.

Reply:
If there are no normal page numbers, write 1-10 if htere are 10 pages, etc.